packages/blade/codemods/brand-refresh/transformers/__tests__/migrate-amount.test.ts

```diff
@@ -57,7 +71,13 @@ it('should migrate the Amount component', async () => {
   expect(result).toMatchInlineSnapshot(`
     "const App = () => (
             <>
-              <Amount value={1234} color="positive" />
+              <Amount value={1234} color="feedback.text.positive.intense" />
```

Comment:
the intent 'positive' is mapped to 'intense' of 'positive' color now?

Reply:
Yes, this is same as before.